src/wolframite/core.clj

```diff
@@ -229,9 +234,49 @@
    (cond-> (convert/convert clj-form (merge {:kernel/link @kernel-link-atom} opts))
      (ifn? output-fn) output-fn)))
 
+(defn load-package!
+  "An extended version of Wolfram's 'Get'. Gets a Wolfram package and makes the constants/functions etc. accessible via a Clojure namespace (the given `alias`, by default the same as `context`).
+
+  Example:  `(<<! \"./resources/WolframPackageDemo.wl\" \"WolframPackageDemo\" 'wp)`
+
+ - `path` - string pointing to the package file
+ - `context` - string for Wolfram context (essentially Wolfram's version of a namespace)
+ - `alias` - Clojure symbol to be used for accessing the Wolfram context. This will effectively become a Clojure namespace
+
+See [[intern-context!]] for details of turning the Wolfram context into a Clojure namespace."
```

Comment:
Should we also have intern-context! in core? If not, then we should change this like this:
```suggestion
See `package/intern-context!` for details of turning the Wolfram context into a Clojure namespace."
```

Reply:
I don't think so. It's not really an everyday kind of function in my view. I'll take the change!